radvd.conf.5.man

```diff
@@ -528,6 +528,14 @@ Note: RFC4861's suggested default value is significantly longer: 7 days.
 
 Default: 14400 seconds (4 hours)
 
+.TP
+.BR AdvSNACRouterFlag " " on | off
+
+When set, indicates the router is a SNAC router (stub network auto
+configuration, e.g., for IPv6-only stub networks).
```

Comment:
Please link to the RFC here

Reply:
I don't think there is an official RFC yet, even though the bit has been assigned (https://github.com/ietf-wg-snac/draft-ietf-snac-simple/issues/54)

Reply:
So add https://www.iana.org/assignments/icmpv6-parameters/icmpv6-parameters.xhtml#icmpv6-parameters-11 and https://datatracker.ietf.org/doc/draft-ietf-6man-snac-router-ra-flag/  which can update when there is an official _RFC_.